src/components/StylePopup/StylePopup.js

```diff
@@ -134,7 +162,10 @@ class StylePopup extends React.PureComponent {
 
 const mapStateToProps = (state, { colorMapKey }) => ({
   currentPalette: selectors.getCurrentPalette(state, colorMapKey),
-  isColorPaletteDisabled: selectors.isElementDisabled(state, 'colorPalette')
+  hideColorPalette: selectors.isElementDisabled(state, DATA_ELEMENTS.COLOR_PALETTE),
+  hideOpacitySlider: selectors.isElementDisabled(state, DATA_ELEMENTS.OPACITY_SLIDER),
+  hideStrokeThicknessSlider: selectors.isElementDisabled(state, DATA_ELEMENTS.STROKE_THICKNESS_SLIDER),
+  hideFontSizeSlider: selectors.isElementDisabled(state, DATA_ELEMENTS.FONT_SIZE_SLIDER),
```

Comment:
Can we use `isSomethingDisabled` as the variable name to match the naming convention in the other components?

Reply:
addressed